ui/components/app/modals/contract-details-modal/contract-details-modal.js

```diff
@@ -65,15 +65,15 @@ export default function ContractDetailsModal({
           fontWeight={FontWeight.Bold}
           variant={TextVariant.bodyMd}
           as="h5"
-          display={DISPLAY.FLEX}
+          display={Display.Flex}
           boxProps={{ marginTop: 0, marginBottom: 0 }}
```

Comment:
hey @georgewrmarshall should this line be removed?

Reply:
Hey @dhruvv173, nice catch the `boxProps` prop should be removed yes. There is no default margin and the Text component does not accept the `boxProps` prop